packages/graphql/tests/schema/issues/2377.test.ts

```diff
@@ -381,12 +382,14 @@ describe("https://github.com/neo4j/graphql/issues/2377", () => {
             }
 
             input ResourceOnCreateInput {
+              createdAt: DateTime!
```

Comment:
This doesn't seem right, this is essentially equivalent to create input, so you shouldn't be able to set this here.